packages/edit-post/src/components/welcome-guide-modal/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Guide from './guide';
+import { CanvasImage, EditorImage, BlockLibraryImage, InserterIconImage } from './vectors';
+
+export default function WelcomeGuideModal() {
+	const areTipsEnabled = useSelect( ( select ) => select( 'core/nux' ).areTipsEnabled() );
```

Comment:
Using this selector from `core/nux` ensures that users who disabled tips will not get the intro modal when updating to the version of Gutenberg that includes this PR. 

Ideally we'd migrate this setting to `core/edit-post`, but I think it's best to do that if/when we remove the `core/nux` package.

Reply:
Let's do this in a follow-up PR.